src/components/pages/LandingPage/LandingPage.js

```diff
@@ -28,12 +28,14 @@ const LandingPage = () => {
             disadvantaged background.
           </p>
           <div className="content-container">
-            <div className="button-container-signin">
-              <h3>Already have an account?</h3>
-              <Button type="primary" onClick={() => loginWithRedirect()}>
-                Log In
-              </Button>
-            </div>
+            {!isAuthenticated /** IF the user is logged in, Hide the button */ ? (
+              <div className="button-container-signin">
+                <h3>Already have an account?</h3>
+                <Button type="primary" onClick={() => loginWithRedirect()}>
+                  Log In
+                </Button>
+              </div>
+            ) : null}
```

Comment:
Great use of a conditional statement! It makes more sense to fix this issue and remove the button once user is logged in.